api/reqs/models.py

```diff
@@ -86,6 +87,7 @@ class Meta:
         max_length=32, choices=[(e.name, e.value) for e in PolicyTypes],
         blank=True
     )
+    slug = models.SlugField(max_length=title.max_length, blank=True)
```

Comment:
What do you think about disallowing blank here? We should expect it to always be set, yes?

Reply:
Yep--I had originally set it to always be set, but then `makemigrations` wanted me to provide a hard-coded default value, which I couldn't provide, since each policy's slug is based on each policy's title. I can think of two options here:

1. Read up on django's docs and see if there's a way to e.g. pass in a function for the default value, so that it can be dynamically set for each policy. This would essentially mean combining the two current migrations into one combined migration.

2. Add a third migration that simply changes `blank=False` and provides a nonsensical default value, since we know from the previous data migration that no blank slugs can possibly exist in the db (at least, not if all the migrations are run in one batch).

Reply:
Option 2 has worked well for me in the past. Option 1 would be tricky due to the need to mix Python and SQL.